spec/app_support/order_row_importer_spec.rb

```diff
@@ -56,7 +55,6 @@
   let(:quantity) { "column4" }
   let(:order_date) { "column5" }
   let(:fulfillment_date) { "column6" }
-  let(:errors) { "column7" }
   let(:notes) { "column8" }
```

Comment:
Notes shouldn't still be column 8, right? I realize this is a string and doesn't really matter, but just for clarity's sake...